packages/compiler-cli/linker/babel/test/ast/babel_ast_factory_spec.ts

```diff
@@ -14,7 +14,7 @@ import {BabelAstFactory} from '../../src/ast/babel_ast_factory';
 
 describe('BabelAstFactory', () => {
   let factory: BabelAstFactory;
-  beforeEach(() => factory = new BabelAstFactory());
+  beforeEach(() => factory = new BabelAstFactory('original.ts'));
```

Comment:
nit: make this path absolute

```suggestion
  beforeEach(() => factory = new BabelAstFactory('/original.ts'));
```